### PR TITLE
Add pcs hget-bnb pool

### PR DESCRIPTION
**Pull request by anecd0che**
**Vault**: 0xf34629872aC0e9152C1E4204b253143e619aFf0A
**Strat**: 0xBDAc328997BD579B49ee01f3AD3707A3bB124E02

### CODE REVIEW

File: src/features/configure/pools.js [hunk 1]
```diff
@@ -23,6 +23,30 @@ export const pools = [
...
+    id: 'cake-hget-bnb',
+    logo: 'cake/hget-bnb.png',
+    name: 'HGET-BNB LP',
+    token: 'HGET-BNB LP',
+    tokenDescription: 'Uses: Pancake',
+    tokenAddress: '0x66b9e1eac8a81f3752f7f3a5e95de460688a17ee',
+    tokenDecimals: 18,
+    tokenDescriptionUrl: '#',
+    earnedToken: 'mooPancakeHGET-BNB',
+    earnedTokenAddress: '0x48Cb3540CB0BB4C6dDF638209e0631Ec85c1a477',
```
Comment (roman-monk) [inline src/features/configure/pools.js:36]: this is mooBDO, should be Vault address `0xf34629872aC0e9152C1E4204b253143e619aFf0A`

**Comment by roman-monk** [comment]
@anecd0che can also add HGET to the filter
https://github.com/beefyfinance/beefy-app/blob/master/src/features/vault/components/Filters/constants.js

File: src/features/configure/pools.js (hunk 1 above)
Comment (anecd0che) [reply src/features/configure/pools.js:36]: Ooh, gotcha, thanks!

File: src/features/configure/pools.js [hunk 2]
```diff
@@ -23,6 +23,30 @@ export const pools = [
     callFee: 0.5,
   },
 
+  {
+    id: 'cake-hget-bnb',
+    logo: 'cake/hget-bnb.png',
```
Comment (roman-monk) [inline src/features/configure/pools.js:28]: should be .svg

**Review by superbeefyboy — APPROVED** [review]

**Pull request merged**